frontend/sass/pages/_home.scss

```diff
@@ -1,4 +1,43 @@
+@use "../abstracts" as abstracts;
+
 .home-content {
-  flex: 1;
-  padding: 40px 60px;
+  padding: 10px 20px;
+  display: grid;
+}
+
+.poster-list {
```

Comment:
Jos sivuun haluaa vähän skaalautuvuutta, tästä voisi tehdä flexin sijaan gridin ja laittaa `grid-template-columns: repeat(6, 1fr)`. Tällöin voi CSS:n lisätä kapeammalle näytölle classista version, jossa gridissä on aina kolme leffaa vierekkäin eikä ne mene niin pieniksi:

```
@media only screen and (max-width: abstracts.$large-width) {
  .poster-list {
    grid-template-columns: repeat(3, 1fr);
  }
}
```